core/os/android/adb/perfetto.go

```diff
@@ -45,7 +46,8 @@ func (b *binding) StartPerfettoTrace(ctx context.Context, config *perfetto_pb.Tr
 
 	reader, stdout := io.Pipe()
 	logRing := ring.New(10)
-	readyOnce := task.Once(ready)
+	// TODO(apbodnar) Find a way to reliably know when Perfetto/producers are ready
```

Comment:
Is there a task for this? It would be good to track this for the next iteration.

Reply:
yep, added it to the TODO